test/db/cmd/cmd_pd2

```diff
@@ -158,16 +153,16 @@ EXPECT=<<EOF
 |       :   0x00560f8a      xchg dword [esp], ebx
 |       :   0x00560f8d      pop esp
 |       :   0x00560f8e      add esp, 4
-|       :   0x00560f91      mov dword [var_ch], eax
-|       :   0x00560f95      mov dword [var_4h], ebx
+|       :   0x00560f91      mov dword [esp + 8], eax
+|       :   0x00560f95      mov dword [esp + 0x10], ebx
 |       :   0x00560f99      push dword [esp]
 |       :   0x00560f9c      pop ebx
 |       :   0x00560f9d      push ebx
 |       :   0x00560f9e      mov ebx, esp
 |       :   0x00560fa0      add ebx, 4
 |       :   0x00560fa6      add ebx, 4
 |       :   0x00560fa9      push ebx
-|       :   0x00560faa      push dword [var_1ch]
+|       :   0x00560faa      push dword [esp + 4]
```

Comment:
is this a regression?

Reply:
Good question. I've added some stuff to the description since the reason basically describes the whole pr.